src/librustdoc/config.rs

```diff
@@ -276,6 +276,8 @@ crate struct RenderOptions {
     crate show_type_layout: bool,
     crate unstable_features: rustc_feature::UnstableFeatures,
     crate emit: Vec<EmitType>,
+    /// If `true`, HTML source pages will generate links for items to their definition.
+    crate generate_link_to_definition: bool,
```

Comment:
They don't exclude each others, so I don't think enums would be helpful here.

Reply:
I meant newtyping the bools for each field, so that someone doesn't accidentally sets `generate_link_to_definition` to true, when they meant to set `show_type_layout`, although with named fields this is less of a concern.

Reply:
Hum... I'm not convinced but I'll open an issue just in case.